### PR TITLE
Add image references

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

**Comment by ironcladlou** [comment]
Should fix the CI errors in https://github.com/openshift/cluster-dns-operator/pull/17

**Comment by ironcladlou** [comment]
@smarterclayton 
```
info: Manifests will be extracted to /tmp/release-image-2018-09-26T214522Z757462613
warning: Could not load current user information: user: unknown userid 1136300000
Loading manifests from cluster-dns-operator: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:8137e6231cbefb67f0f8fa65bd8b3911317162dff6dca744a1c87d8ef6e90de1 ...
Loading manifests from catalog: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:bf9e1b9e0acbb056484a51fefe9dffb23bb7184e214cb48c344924fbe7f90091 ...
Loading manifests from cluster-image-registry-operator: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:df10457dcee04dbea079253a67d97c233ef621ef0dd939c8a9bca03e849de846 ...
Loading manifests from cluster-version-operator: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:6ed371948bfaf746dcb0263176687a625b1ef581447527cbb840b69c4aa7fa29 ...
Loading manifests from cluster-openshift-controller-manager-operator: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:e34d27854162ac61e4a484c517651f5bd626c21d2dad126f91fe4ecd901f4529 ...
Loading manifests from cluster-openshift-apiserver-operator: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:770c03f88d9a947c9a1d3f29c5a0e313031dde86052fc5a151a99314e9d4fe6e ...
Loading manifests from cluster-samples-operator: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:bd3b11dacb5c4524dcd1e5c2814681174227d1b3e015aee99183e8fcd9d77fcc ...
Loading manifests from cluster-kube-apiserver-operator: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:12ba06fada065a12f8d2c3be27dcfa6577cb0a911886a5958a373f9e5d97b26f ...
Loading manifests from cluster-kube-controller-manager-operator: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:0ce689191e2337e62f460a328f1aaf7d1ac12a80760793f73f96be7f7e5932b1 ...
Loading manifests from machine-config-operator: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:d2c602936193f81a808d477b065c01dab18d15914b2ec71c380f310d4a44b441 ...
Loading manifests from olm: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:43d39c9645cf54594f06fccd36a2ee294a9e6f7b4e09956f50fa1410a868e11e ...
Loading manifests from service-serving-cert-signer: registry.svc.ci.openshift.org/ci-op-mykxqgpz/stable@sha256:e455472342e6554067046dbe6cd277f5fd0587cfaf56d21eaa54439b770439a5 ...
error: unable to upload new layer (0): Patch https://registry.svc.ci.openshift.org/v2/ci-op-mykxqgpz/release/blobs/uploads/1d18b7e7-92f0-494b-85f6-46c3475bfe76?_state=rGB-JaAkjvajXr6zFVXuxz-uH6OEk_aiw2HrSWTmRS17Ik5hbWUiOiJjaS1vcC1teWt4cWdwei9yZWxlYXNlIiwiVVVJRCI6IjFkMThiN2U3LTkyZjAtNDk0Yi04NWY2LTQ2YzM0NzViZmU3NiIsIk9mZnNldCI6MCwiU3RhcnRlZEF0IjoiMjAxOC0wOS0yNlQyMTo0NTozMC45NjczOTA4MDlaIn0%3D: operator "catalog" failed to map images: yaml: line 16: mapping values are not allowed in this context
Uploading ... 2018/09/26 21:45:33 Container release in pod release-latest failed, exit code 1, reason Error
```

This looks like some `catalog` manifest issue which is breaking our builds?

File: manifests/image-references [hunk 1]
```diff
@@ -0,0 +1,6 @@
+kind: ImageStream
+apiVersion: v1
+  spec:
+    tags:
+    - name: cluster-dns-operator
+      from: {"kind": "DockerImage", "name": "openshift/origin-cluster-dns-operator:latest"}
```
Comment (abhinavdahiya) [inline manifests/image-references]: This should be 
```yaml
kind: ImageStream
apiVersion: v1
  spec:
    tags:
    - name: cluster-dns-operator
      from: 
        kind: "DockerImage"
        name: "openshift/origin-cluster-dns-operator:latest"
```

Comment (ironcladlou) [reply manifests/image-references]: Reformatted

**Comment by ironcladlou** [comment]
@abhinavdahiya 
>Also seems like service account is next in order to deployment (https://github.com/openshift/cluster-dns-operator/tree/master/manifests), The deployment will not complete rollout with out an existing service account and ClusterVersionOperator waits for rollout to complete for a deployment as its done criteria. Do you mind changing the order to serviceaccount then deployment.

Thanks, will address this in a followup.

**Pull request merged**